embulk-input-sqlserver/build.gradle

```diff
@@ -11,3 +11,38 @@ embulkPlugin {
     category = "input"
     type = "sqlserver"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Shintaro Kimura"
+                        email = "kimura@enigmo.co.jp"
+                    }
```

Comment:
@kmrshntr Hi!

We're going to upload `embulk-input-jdbc` (including `embulk-input-sqlserver`) to Maven Central, and Maven Central needs a `<developers>` section in `pom.xml`. On this occasion, we're trying to nominate past important contributors in the `<developers>` section.

We wonder if you're okay to nominate you there on Maven Central, from your work in #137 and #139. I picked up your name and email from the Git commit log, which are already public.

Please let me know if you don't want, or if you want another name/email.

Reply:
I am honored to be nominated. The name is OK. Please use my personal email address kmrshntr[at]gmail.com. 
Thank you.

Reply:
Thanks! We're happy to have you there. Updated the email in 69525862759f2807784cfccfbf0b64b0f0e27bdc.